common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -979,6 +979,11 @@
           "The <paramIndex> parameter requires the <requiredType> type, however <inputSql> has the type <inputType>."
         ]
       },
+      "UNSUPPORTED_MODE_DATA_TYPE" : {
+        "message" : [
+          "The <mode> mode does not support the <child> data type, because <reason>."
```

Comment:
Regarding the beginning of the error message, how the final text looks like:
The 'mode mode
?
If so, let's change it.